frontend/src/store/modules/searchResults.js

```diff
@@ -64,6 +64,9 @@ const actions = {
         results.push(JSON.parse(filteredResults[item].document))
       }
     }
+    results.sort(function (a, b) {
+      return a.name.localeCompare(b.name, 'da')
```

Comment:
Can't SOLR do this for you?

Reply:
Good question. I'll look into it.

Reply:
It requires more work than it's worth. Sorting in UI is efficient enough. I've added comments with explanations.